staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -199,9 +199,10 @@ type Routes interface {
 }
 
 var (
-	InstanceNotFound = errors.New("instance not found")
-	DiskNotFound     = errors.New("disk is not found")
-	NotImplemented   = errors.New("unimplemented")
+	DiskNotFound         = errors.New("disk is not found")
+	ImplementedElsewhere = errors.New("implemented by a different controller")
```

Comment:
nit: Your use case is an alternate controller but I think we might want to be more general. "implemented by alternate to cloud provider"?